### DISCUSSION TITLE
Initial installation of extension

### DISCUSSION THREAD

**Discussion by Syl100** [Cookie-AutoDelete/Cookie-AutoDelete#1214] [Q&A / Support] [answered]
What happens to existing cookies when the extension is first installed - are they deleted?

**Comment by kennethtran93** [comment, marked as answer]
The extension does not clean any cookies upon first installation, as the auto-clean option is disabled by default.

See: https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/wiki/FAQ:-Common-Questions-and-Issues#why-is-automatic-cleaning-off-by-default
